.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: integration-tests
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8' ]
+        architecture: [ 'x64' ]
+    name: Integration tests for CLI
+    steps:
+      - uses: actions/checkout@v2.3.2
+      - name: Setup JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+          architecture: ${{ matrix.architecture }}
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+
+      - name: Make
+        run: cd distro && make
+
+      - name: Run CLI for IMDG
+        run: distro/build/dist/bin/hz start > log-imdg.txt 2>&1 &
+
+      - name: Check output
+        run: sleep 15s && cat log-imdg.txt | grep "Members {size:1"
```

Comment:
Sleeping for 15s is a lot, what about just retrying every second a couple of times?

https://github.com/hazelcast-guides/hazelcast-client-quarkus/blob/master/.github/workflows/build.yml#L36-L48

Reply:
yes, this will be better 👍